thrust/thrust/functional.h

```diff
@@ -984,7 +974,7 @@ THRUST_BINARY_FUNCTOR_VOID_SPECIALIZATION(maximum, t1 < t2 ? THRUST_FWD(t2) : TH
  *  \see binary_function
  */
 template <typename T = void>
-struct minimum
+struct minimum : cuda::std::minimum<T>
```

Comment:
That should be `cuda::minimum`, not `cuda::std::minimum`.

Reply:
fixed